src/shared/locales/ptBr/translations/incidents/index.ts

```diff
@@ -0,0 +1,44 @@
+export default {
+  incidents: {
+    filterTitle: 'Filtrar por status',
+    label: 'Incidentes',
+    actions: {
+      report: 'Relatório',
+    },
+    status: {
+      reported: 'Criado',
+      resolved: 'Resolvido',
+      all: 'All',
+    },
+    reports: {
+      label: 'Incidentes criados',
+      new: 'Criar Incidente',
+      view: 'Ver Incidente',
+      resolve: 'Resolver Incidente',
+      dateOfIncident: 'Data do Incidente',
+      department: 'Departmento',
```

Comment:
Typo
```suggestion
      department: 'Departamento',
```

Reply:
Departament is Departamento in portguese

![image](https://user-images.githubusercontent.com/5497267/135870773-5a4265ae-686d-4c83-ac72-25a3d88b175f.png)

Reply:
exactly, it's missing an a between the t and the m 😉